libvast/src/detail/installdirs.cpp

```diff
@@ -0,0 +1,57 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2021 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "vast/detail/installdirs.hpp"
+
+#include "vast/config.hpp"
+#include "vast/detail/assert.hpp"
+#include "vast/detail/process.hpp"
+
+namespace vast::detail {
+
+namespace {
+
+std::filesystem::path install_prefix() {
+  const auto binary = detail::objectpath(nullptr);
+  VAST_ASSERT(binary);
+  return binary->parent_path().parent_path();
+}
+
+} // namespace
+
+std::filesystem::path install_datadir([[maybe_unused]] bool bare_mode) {
+#if VAST_ENABLE_RELOCATABLE_INSTALLATIONS
+  return install_prefix() / VAST_DATADIR;
+#else
+  if (!bare_mode)
```

Comment:
```suggestion
  if (bare_mode)
```

Reply:
TODO: Remove the bare mode fallback, change integration tests for non-relocatable builds to use VAST from CMAKE_INSTALL_FULL_BINDIR.